extra/exports

```diff
@@ -100,6 +100,7 @@ box_region_used
 box_replace
 box_return_mp
 box_return_tuple
+box_request_process
```

Comment:
> But it's still not clear to me if box_request_process is necessary.

It doesn't matter if it is necessary, speaking in terms of patchset structure. Such huge refactoring should go in a separate preparatory commit. In a first commit you introduce it, change all the needed places to start using `box_request_process`. In the second commit you add your new flag (which shouldn't be needed) and the meta-spaces.

Currently `box_request_process` introduction is even bigger than the meta-spaces feature. Hard to see which changes relate to this new function and which to your new feature.

Reply:
Sorry about that

Reply:
Here you go